directory/cli/src/group.py

```diff
@@ -91,7 +104,82 @@ def remove_member(group_name, users):
         except IpaRunError:
             _diagnose_member_command_error(group_name, users, add_command=False)
 
-    wrapper_commands = [
+    if not utils.advanced_mode_enabled():
+        @group.command(name='create', help='Create a new group')
+        def create():
+            click.echo('Please enter details for the following:')
+
+            params = {
+                'name': click.prompt('  Group name'),
+                'desc': click.prompt('  Description')
+            }
+            if click.confirm(
+                'Would you like to set a specific GID for this group? (Default: Autogenerated GID)'
+            ):
+                params = { **params, 'gid': click.prompt('  GID') }
+
+            wrapper = ipa_wrapper_command.create_ipa_wrapper(
+                'group-add',
+                argument_name='name',
+                transform_options_callback=_validate_blacklist_groups
+            )
+            wrapper(**params)
+
+        @group.command(name='modify', help='Modify an existing group')
+        def modify():
+            click.echo('Please enter the name of the group you want to modify:')
+            group = click.prompt('  Group name')
+            group_find_args = ['--group-name={}'.format(group)]
+            try:
+                group_data = ipa_utils.ipa_find(
+                        'group-find',
+                        group_find_args,
+                        all_fields=True
+                )[0]
+            except IpaRunError:
+                error = '{}: group not found'.format(group)
+                raise click.ClickException(error)
+
+            _validate_blacklist_groups(group)
+
+            click.echo(
+                'Adjust the following fields as necessary:\n'
+                'Leave blank to keep current value shown within brackets'
+            )
+            params = {
+                'name': group,
+                'desc': click.prompt(
+                    '  Description',
+                    default=group_data['Description'][0]
+                )
+            }
+
+            wrapper = ipa_wrapper_command.create_ipa_wrapper(
+                'group-mod',
+                argument_name='name',
+            )
+            wrapper(**params)
+
+        @member.command(name='add', help='Add user(s) to a group')
+        def add_member():
+            click.echo('Please enter the name of the group you wish to add to:')
+            group = click.prompt('  Group name')
+            _validate_blacklist_groups(group)
+
+            click.echo(
+                'Please enter the user(s) you wish to add to %s, separated by spaces:' % group
+            )
+            users = click.prompt('  User(s)').split()
+            user_options = ['--users={}'.format(user) for user in users]
+
+            args = [group] + user_options
+            try:
+                ipa_utils.ipa_run('group-add-member', args, error_in_stdout=True)
+                utils.display_success()
```

Comment:
A couple of things to mention about this:
1. I know you've just stuck with doing similar to what we did in the advanced mode `member` commands here, but I've not seen those before so thought I'd mention now - is it/why is it needed to display success for these commands? This seems slightly inconsistent with our other commands where we don't do this; also typically CLI tools tend to be fairly quiet by default when the run was successful, since this is what the user expects to happen and this makes errors/other important output stand out more. Possibly @DavidMarchant may know why we originally added this.
2. If we do want these lines, given https://github.com/alces-software/userware/blob/980ea7f495886fc032b14d584d11f3bbaf0602b3/directory/cli/src/utils.py#L107 and the member commands are now in a new `member` group, I believe this means this will print something like `group member successful` (because `" ".join(['group', 'member', 'add'][:2]) == 'group member'`), whereas we would want it to print `group member add` successful.

    If we did want to fix this, I don't think we can just change this to `[:3]` as this would include too much in other cases where this function is used - we can fully investigate how to handle this if we decide we need to keep it; I'd imagine doing this properly would involve recursively finding all the parent commands for the currently running command, but maybe there's an easier way to find this out.

Reply:
As for point 1 I believe @mjtko wanted some more feedback on command completion 
The relevant issue - though not detailed - is here https://github.com/alces-software/userware/issues/43.

I have no idea if this is still desired. 
(and if there are commands where this doesn't happen it should certainly be all-or-none)

Reply:
Alright cheers for getting back to us. If Mark is happy for us to remove them then I will make another PR soon™ that does so. Otherwise I will leave them for now.

Reply:
After some discussion it was clarified that we do indeed want to keep these messages, for now at least. Userware is only available via a sandbox currently and this is unlikely to change in the immediate future so having feedback can be useful.